### ISSUE TITLE
Only partial list of files available

### ISSUE THREAD

**Issue by worstofthewurst** [Hashirama/OFDL#32]
Everything works as it should, except that only a partial list of files are linked and downloadable. For example, one user lists 209 videos on their profile, and yet only 63 are linked and downloadable.

Is this a known issue?

**Comment by Alexo88** [comment]
some of the videos are paid, are you sure that all the videos are free?

**Comment by ensozero** [comment]
Yes, I'm completely sure. This user has zero paid content, and OFDL recognizes 209 but downloads 62.

**Comment by pbradjr99** [comment]
If you don't mind who is the username, is it a paid account? I'll try another downloader I have and see what it pulls.  Just to make sure it's not this download program

**Comment by Alexo88** [comment]
you can try with SCrawler, maybe you won't get the same, the fact that it doesn't download all the videos is normal in several repositories I tried.

**Comment by ensozero** [comment]
I installed Microsoft Visual Studio to run SCrawler but I'm absolutely hopeless about getting this up and running. The instructions assume you know a visual basic environment, and I don't.

**Comment by someguy75** [comment]
It seems that the post API only returns 50 results when called with a limit of 100. The subsequent offsets are calculated assuming that the full amount is returned so some will be skipped.

https://github.com/Hashirama/OFDL/blob/36bc39c9b2aed1f4278b4d99182d26532ed068c2/module/snafylno.py#L791-L800

**Comment by Hashirama** [comment]
Ya I'll look into it.

**Comment by longusnickus** [comment]
Some videos have DRM Protection. It shows the right amount of videos, but they cannot be downloaded. 
In Firefox: Shift+Right Click on a Video/Image
You can download all media like that, execpt when a video has DRM
